.github/workflows/labeler.yml

```
@@ -0,0 +1,12 @@
+name: Labeler
+
+on:
+  - pull_request_target
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/labeler@v4.3.0
+        with:
+          repo-token: ${{ secrets.GH_TOKEN }}
```

```suggestion
          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

This should be fine:
- https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication
- https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#permissions-for-the-github_token